OSX CI intermittent failures

Seems to depend on what machine we get.  Failures look like:
```
2020-01-16T16:27:28.2591350Z   [ 98%] Built target daccess
2020-01-16T16:27:28.2591630Z   Scanning dependencies of target mscordaccore
2020-01-16T16:27:28.2592530Z   [ 98%] Building CXX object src/dlls/mscordac/CMakeFiles/mscordaccore.dir/__/__/__/version.cpp.o
2020-01-16T16:27:28.2592870Z   [ 98%] Linking CXX shared library libmscordaccore.dylib
2020-01-16T16:27:28.2593240Z   [ 98%] Built target mscordaccore
2020-01-16T16:27:28.2593540Z   Scanning dependencies of target mscordbi
2020-01-16T16:27:28.2593900Z   [ 98%] Building CXX object src/dlls/mscordbi/CMakeFiles/mscordbi.dir/mscordbi.cpp.o
2020-01-16T16:27:28.2594210Z   [ 98%] Building CXX object src/dlls/mscordbi/CMakeFiles/mscordbi.dir/__/__/__/version.cpp.o
2020-01-16T16:27:28.2594500Z   [ 99%] Linking CXX shared library libmscordbi.dylib
2020-01-16T16:27:28.2594870Z   Scanning dependencies of target sos
2020-01-16T16:27:28.2595160Z   [ 99%] Building CXX object src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/eeheap.cpp.o
2020-01-16T16:27:28.2595540Z   [ 99%] Building CXX object src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/disasm.cpp.o
2020-01-16T16:27:28.2595840Z   [ 99%] Building CXX object src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/datatarget.cpp.o
2020-01-16T16:27:28.2596210Z   [ 99%] Building CXX object src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/exts.cpp.o
2020-01-16T16:27:28.2596670Z   [ 99%] Built target mscordbi
2020-01-16T16:27:28.2598640Z   [100%] Building CXX object src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/gchist.cpp.o
2020-01-16T16:27:28.2598990Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/datatarget.cpp:5:
2020-01-16T16:27:28.2599310Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/sos.h:7:
2020-01-16T16:27:28.2599570Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/strike.h:109:
2020-01-16T16:27:28.2599980Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/exts.h:81:
2020-01-16T16:27:28.2600340Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/util.h:2775:
2020-01-16T16:27:28.2600590Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/unordered_map:408:
2020-01-16T16:27:28.2601450Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__hash_table:18:
2020-01-16T16:27:28.2601870Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:645:
2020-01-16T16:27:28.2603240Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1493:31: error: expected member name or ';' after declaration specifiers
2020-01-16T16:27:28.2603760Z       const _Alloc& __allocator() const { return __f_.second(); }
2020-01-16T16:27:28.2604080Z       ~~~~~~~~~~~~              ^
2020-01-16T16:27:28.2604950Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1615:29: error: expected unqualified-id
2020-01-16T16:27:28.2605430Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2605720Z                               ^
2020-01-16T16:27:28.2606690Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1626:57: error: expected unqualified-id
2020-01-16T16:27:28.2607180Z       ::new (__p) __func(__f_.__target(), __f_.__allocator());
2020-01-16T16:27:28.2607580Z                                                           ^
2020-01-16T16:27:28.2608470Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1642:29: error: expected unqualified-id
2020-01-16T16:27:28.2609200Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2609500Z                               ^
2020-01-16T16:27:28.2610570Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1928:39: error: expected unqualified-id
2020-01-16T16:27:28.2611410Z           _FunAlloc __a(__f->__allocator());
2020-01-16T16:27:28.2611890Z                                         ^
2020-01-16T16:27:28.2612220Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/exts.cpp:10:
2020-01-16T16:27:28.2612540Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/exts.h:81:
2020-01-16T16:27:28.2612940Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/util.h:2775:
2020-01-16T16:27:28.2613250Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/unordered_map:408:
2020-01-16T16:27:28.2613670Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__hash_table:18:
2020-01-16T16:27:28.2614060Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:645:
2020-01-16T16:27:28.2615010Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1493:31: error: expected member name or ';' after declaration specifiers
2020-01-16T16:27:28.2615500Z       const _Alloc& __allocator() const { return __f_.second(); }
2020-01-16T16:27:28.2615820Z       ~~~~~~~~~~~~              ^
2020-01-16T16:27:28.2616680Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1615:29: error: expected unqualified-id
2020-01-16T16:27:28.2617160Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2617450Z                               ^
2020-01-16T16:27:28.2618410Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1626:57: error: expected unqualified-id
2020-01-16T16:27:28.2618800Z       ::new (__p) __func(__f_.__target(), __f_.__allocator());
2020-01-16T16:27:28.2619300Z                                                           ^
2020-01-16T16:27:28.2620530Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1642:29: error: expected unqualified-id
2020-01-16T16:27:28.2621020Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2621330Z                               ^
2020-01-16T16:27:28.2621700Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/eeheap.cpp:11:
2020-01-16T16:27:28.2622040Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/sos.h:7:
2020-01-16T16:27:28.2622430Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/strike.h:109:
2020-01-16T16:27:28.2622760Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/exts.h:81:
2020-01-16T16:27:28.2623160Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/util.h:2775:
2020-01-16T16:27:28.2623510Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/unordered_map:408:
2020-01-16T16:27:28.2623910Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__hash_table:18:
2020-01-16T16:27:28.2624250Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:645:
2020-01-16T16:27:28.2625520Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1493:31: error: expected member name or ';' after declaration specifiers
2020-01-16T16:27:28.2625930Z       const _Alloc& __allocator() const { return __f_.second(); }
2020-01-16T16:27:28.2626320Z       ~~~~~~~~~~~~              ^
2020-01-16T16:27:28.2627200Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1615:29: error: expected unqualified-id
2020-01-16T16:27:28.2627780Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2628120Z                               ^
2020-01-16T16:27:28.2628990Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1626:57: error: expected unqualified-id
2020-01-16T16:27:28.2629470Z       ::new (__p) __func(__f_.__target(), __f_.__allocator());
2020-01-16T16:27:28.2629770Z                                                           ^
2020-01-16T16:27:28.2630730Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1642:29: error: expected unqualified-id
2020-01-16T16:27:28.2631120Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2631500Z                               ^
2020-01-16T16:27:28.2632490Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1928:39: error: expected unqualified-id
2020-01-16T16:27:28.2633410Z           _FunAlloc __a(__f->__allocator());
2020-01-16T16:27:28.2633780Z                                         ^
2020-01-16T16:27:28.2634740Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1928:39: error: expected unqualified-id
2020-01-16T16:27:28.2635580Z           _FunAlloc __a(__f->__allocator());
2020-01-16T16:27:28.2636050Z                                         ^
2020-01-16T16:27:28.2636370Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/disasm.cpp:11:
2020-01-16T16:27:28.2636570Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/strike.h:109:
2020-01-16T16:27:28.2636970Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/exts.h:81:
2020-01-16T16:27:28.2637530Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/util.h:2775:
2020-01-16T16:27:28.2638090Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/unordered_map:408:
2020-01-16T16:27:28.2638420Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__hash_table:18:
2020-01-16T16:27:28.2638850Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:645:
2020-01-16T16:27:28.2639780Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1493:31: error: expected member name or ';' after declaration specifiers
2020-01-16T16:27:28.2640350Z       const _Alloc& __allocator() const { return __f_.second(); }
2020-01-16T16:27:28.2640700Z       ~~~~~~~~~~~~              ^
2020-01-16T16:27:28.2641910Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1615:29: error: expected unqualified-id
2020-01-16T16:27:28.2642310Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2642710Z                               ^
2020-01-16T16:27:28.2643580Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1626:57: error: expected unqualified-id
2020-01-16T16:27:28.2644280Z       ::new (__p) __func(__f_.__target(), __f_.__allocator());
2020-01-16T16:27:28.2644600Z                                                           ^
2020-01-16T16:27:28.2645550Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1642:29: error: expected unqualified-id
2020-01-16T16:27:28.2645940Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2646340Z                               ^
2020-01-16T16:27:28.2647220Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1928:39: error: expected unqualified-id
2020-01-16T16:27:28.2648220Z           _FunAlloc __a(__f->__allocator());
2020-01-16T16:27:28.2648600Z                                         ^
2020-01-16T16:27:28.2648910Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/gchist.cpp:36:
2020-01-16T16:27:28.2718400Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/strike.h:109:
2020-01-16T16:27:28.2718610Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/exts.h:81:
2020-01-16T16:27:28.2719040Z   In file included from /Users/runner/runners/2.163.1/work/1/s/src/coreclr/src/ToolBox/SOS/Strike/util.h:2775:
2020-01-16T16:27:28.2719240Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/unordered_map:408:
2020-01-16T16:27:28.2719470Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__hash_table:18:
2020-01-16T16:27:28.2719720Z   In file included from /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:645:
2020-01-16T16:27:28.2720950Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1493:31: error: expected member name or ';' after declaration specifiers
2020-01-16T16:27:28.2721390Z       const _Alloc& __allocator() const { return __f_.second(); }
2020-01-16T16:27:28.2721690Z       ~~~~~~~~~~~~              ^
2020-01-16T16:27:28.2722550Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1615:29: error: expected unqualified-id
2020-01-16T16:27:28.2723320Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2723610Z                               ^
2020-01-16T16:27:28.2724510Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1626:57: error: expected unqualified-id
2020-01-16T16:27:28.2724900Z       ::new (__p) __func(__f_.__target(), __f_.__allocator());
2020-01-16T16:27:28.2725220Z                                                           ^
2020-01-16T16:27:28.2726090Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1642:29: error: expected unqualified-id
2020-01-16T16:27:28.2726490Z       _Ap __a(__f_.__allocator());
2020-01-16T16:27:28.2726780Z                               ^
2020-01-16T16:27:28.2727650Z   /Applications/Xcode_11.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:1928:39: error: expected unqualified-id
2020-01-16T16:27:28.2728480Z           _FunAlloc __a(__f->__allocator());
2020-01-16T16:27:28.2728850Z                                         ^
2020-01-16T16:27:28.2729140Z   5 errors generated.
2020-01-16T16:27:28.2729420Z   make[2]: *** [src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/datatarget.cpp.o] Error 1
2020-01-16T16:27:28.2729730Z   make[2]: *** Waiting for unfinished jobs....
2020-01-16T16:27:28.2730010Z   5 errors generated.
2020-01-16T16:27:28.2730300Z   make[2]: *** [src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/exts.cpp.o] Error 1
2020-01-16T16:27:28.2730890Z   5 errors generated.
2020-01-16T16:27:28.2731190Z   make[2]: *** [src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/eeheap.cpp.o] Error 1
2020-01-16T16:27:28.2731490Z   5 errors generated.
2020-01-16T16:27:28.2731760Z   make[2]: *** [src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/disasm.cpp.o] Error 1
2020-01-16T16:27:28.2732120Z   5 errors generated.
2020-01-16T16:27:28.2732410Z   make[2]: *** [src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/gchist.cpp.o] Error 1
2020-01-16T16:27:28.2732720Z   make[1]: *** [src/ToolBox/SOS/Strike/CMakeFiles/sos.dir/all] Error 2
2020-01-16T16:27:28.2733010Z   make: *** [all] Error 2
2020-01-16T16:27:28.2733300Z   Failed to build CoreCLR component.
2020-01-16T16:27:28.2733580Z   
2020-01-16T16:27:28.2734360Z   'coreclr' failed during build.
2020-01-16T16:27:28.2735240Z   See '/Users/runner/runners/2.163.1/work/1/s/bin/logs/coreclr.log' for more information.
2020-01-16T16:27:28.2736040Z   'known-good' failed during build.
2020-01-16T16:27:28.2737690Z ##[debug]Exit code 1 received from tool '/bin/bash'
2020-01-16T16:27:28.2738810Z ##[debug]STDIO streams have closed for tool '/bin/bash'
2020-01-16T16:27:28.2827230Z ##[error]Bash exited with code '1'.
2020-01-16T16:27:28.2841400Z ##[debug]Processed: ##vso[task.issue type=error;]Bash exited with code '1'.
2020-01-16T16:27:28.2842120Z ##[debug]task result: Failed
2020-01-16T16:27:28.2844190Z ##[debug]Processed: ##vso[task.complete result=Failed;done=true;]
2020-01-16T16:27:28.2846050Z ##[section]Finishing: Build source-build
```

--- a/.vsts.pipelines/steps/setup-macos-native.yml
+++ b/.vsts.pipelines/steps/setup-macos-native.yml
@@ -9,10 +9,16 @@ steps:
     }
     set -x
     # Work around brew bug: https://github.com/dotnet/coreclr/pull/21913
+    # Update to openSSL1.1 due to https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl
     brew update
     # Install macOS native dependencies not present in hosted pool.
     # Based on https://github.com/sbomer/coreclr/blob/dc2fa1b96415f87d694e5b6e751e38ca6fbd09a1/unix-pipeline.yml#L36-L39
+
     brew install icu4c openssl
+
+    # You might need to "link" pkg-config:
+    brew link pkg-config
+
     brew link --force icu4c
     # Follow https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md#user-content-macos
     brew install cmake pkgconfig openssl
@@ -20,4 +26,27 @@ steps:
     # Use 'for' to get last/latest openssl install dir.
     for openssl_path in $brew_openssl_paths; do :; done
     setvariable OPENSSL_ROOT_DIR "$openssl_path"
+
+    # We need to make the runtime libraries discoverable, as well as make
+    # pkg-config be able to find the headers and current ABI version.
+    #
+    # Ensure the paths we will need exist
+    mkdir -p /usr/local/lib/pkgconfig
+
+    # The rest of these instructions assume a default Homebrew path of
+    # `/usr/local/opt/<module>`, with `brew --prefix` returning `/usr/local`
+    # and `brew --prefix openssl` returning `/usr/local/opt/openssl@1.1`.
+    # In this case, `brew info openssl` shows:
+    # `openssl@1.1: stable 1.1.1d (bottled) [keg-only]`.
+
+    # Runtime dependencies
+    ln -s /usr/local/opt/openssl\@1.1/lib/libcrypto.1.1.dylib /usr/local/lib/
+    ln -s /usr/local/opt/openssl\@1.1/lib/libssl.1.1.dylib /usr/local/lib/
+
+    # Compile-time dependencies (for pkg-config)
+    ln -s /usr/local/opt/openssl\@1.1/lib/pkgconfig/libcrypto.pc /usr/local/lib/pkgconfig/
+    ln -s /usr/local/opt/openssl\@1.1/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
+    ln -s /usr/local/opt/openssl\@1.1/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
+
+
   displayName: Install native dependencies

--- a/.vsts.pipelines/steps/setup-macos-native.yml
+++ b/.vsts.pipelines/steps/setup-macos-native.yml
@@ -8,24 +8,26 @@ steps:
       )
     }
     set -x
+
     # Work around brew bug: https://github.com/dotnet/coreclr/pull/21913
-    # Update to openSSL1.1 due to https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl
     brew update
+
     # Install macOS native dependencies not present in hosted pool.
     # Based on https://github.com/sbomer/coreclr/blob/dc2fa1b96415f87d694e5b6e751e38ca6fbd09a1/unix-pipeline.yml#L36-L39
-
     brew install icu4c openssl
-
-    # You might need to "link" pkg-config:
-    brew link pkg-config
-
     brew link --force icu4c
+
     # Follow https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md#user-content-macos
     brew install cmake pkgconfig openssl
     brew_openssl_paths=$(brew info openssl | cut -d ' ' -f 1 | grep '^/.*/openssl/')
+
     # Use 'for' to get last/latest openssl install dir.
     for openssl_path in $brew_openssl_paths; do :; done
     setvariable OPENSSL_ROOT_DIR "$openssl_path"
+
+    # Update to openSSL1.1 due to https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl
+    # "link" pkg-config:
+    brew link pkg-config
 
     # We need to make the runtime libraries discoverable, as well as make
     # pkg-config be able to find the headers and current ABI version.

--- a/patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch
+++ b/patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch
@@ -1,7 +1,7 @@
-From f2a4abfda4f2dea0d1bece590222b304b6242cf3 Mon Sep 17 00:00:00 2001
+From 015b1c02751e2ff0a28397dd03183befdee847db Mon Sep 17 00:00:00 2001
 From: adaggarwal <aditya.aggarwal@microsoft.com>
 Date: Fri, 6 Mar 2020 18:25:45 +0000
-Subject: [PATCH] glibc sysctl deprecation handling
+Subject: [PATCH 1/2] glibc sysctl deprecation handling
 
 ---
  src/pal/src/misc/sysinfo.cpp | 7 +++++--

--- a/patches/coreclr/0002-clang9-resolution-to-conditionally-include-allocator.patch
+++ b/patches/coreclr/0002-clang9-resolution-to-conditionally-include-allocator.patch
@@ -1,0 +1,27 @@
+From 2b5e1784bdd399e5c1112935546757f1b0542326 Mon Sep 17 00:00:00 2001
+From: Aditya Aggarwal <aditya.aggarwal@microsoft.com>
+Date: Fri, 27 Mar 2020 19:18:42 +0000
+Subject: [PATCH 2/2] clang9 resolution to conditionally include allocator
+ header file
+
+---
+ src/pal/inc/rt/specstrings.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/pal/inc/rt/specstrings.h b/src/pal/inc/rt/specstrings.h
+index 214d4190de..93289c87f7 100644
+--- a/src/pal/inc/rt/specstrings.h
++++ b/src/pal/inc/rt/specstrings.h
+@@ -310,8 +310,8 @@ __ANNOTATION(SAL_failureDefault(enum __SAL_failureKind));
+ 				            __byte_readableTo((expr) ? (size) : (size) * 2)
+ #define __post_invalid                      _Post_ __notvalid
+ /* integer related macros */
+-#define __allocator                         __inner_allocator
+ #ifndef PAL_STDCPP_COMPAT
++#define __allocator                         __inner_allocator
+ #define __deallocate(kind)                  _Pre_ __notnull __post_invalid
+ #define __deallocate_opt(kind)              _Pre_ __maybenull __post_invalid
+ #endif
+-- 
+2.21.0
+

--- a/patches/coreclr/0002-clang9-resolution-to-conditionally-include-allocator.patch
+++ b/patches/coreclr/0002-clang9-resolution-to-conditionally-include-allocator.patch
@@ -4,6 +4,7 @@ Date: Fri, 27 Mar 2020 19:18:42 +0000
 Subject: [PATCH 2/2] clang9 resolution to conditionally include allocator
  header file
 
+Issue link: https://github.com/dotnet/source-build/issues/1449
 ---
  src/pal/inc/rt/specstrings.h | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)

